docs/section11/Rule_11-10.md

```diff
@@ -0,0 +1,221 @@
+        - check whether the uniform_energy_factor is valid: `if uniform_energy_factor != NULL
+          - check if the actual uniform_energy_factor is >= to the expected: `if uniform_energy_factor >= expected_UEF:`
+            - set rule_status to "PASS": `rule_status = "PASS"`
+          - otherwise, set rule status to FAIL and provide the note that the expected UEF is less than the required UEF: `else: rule_status = "FAIL"; rule_note = "The modeled Uniform energy Factor (UEF - " + uniform_energy_factor + ") is less than required (" + expected_UEF + ")"`
+        - otherwise, set rule status to UNDETERMINED and provide a note: `else: rule_status = "UNDETERMINED"; rule_note = "Uniform Energy Factor is required but not provided"`
+      - else, if the storage tank is is between 20 and 55 gallons, this is considered an electric storage water heater in Appendix F: `elif storage_volume >= 20 && storage_volume <= 55:`
+        - determine the UEF based on the draw pattern for VERY_SMALL: `if draw_pattern == "VERY_SMALL": expected_uef = 0.8808 – (0.0008 × storage_volume)`
+        - determine the UEF based on the draw pattern for LOW: `if draw_pattern == "LOW": expected_uef = 0.9254 – (0.0003 × storage_volume)`
+        - determine the UEF based on the draw pattern for MEDIUM: `if draw_pattern == "MEDIUM": expected_uef = 0.9307 – (0.0002 × storage_volume)`
+        - determine the UEF based on the draw pattern for HIGH: `if draw_pattern == "HIGH": expected_uef = 0.9349 – (0.0001 × storage_volume)`
+        - check whether the uniform_energy_factor is valid: `if uniform_energy_factor != NULL
+          - check if the actual uniform_energy_factor is >= to the expected: `if uniform_energy_factor >= expected_UEF:`
+            - set rule_status to "PASS": `rule_status = "PASS"`
+          - otherwise, set rule status to FAIL and provide the note that the expected UEF is less than the required UEF: `else: rule_status = "FAIL"; rule_note = "The modeled Uniform energy Factor (UEF - " + uniform_energy_factor + ") is less than required (" + expected_UEF + ")"`
+        - otherwise, set rule status to UNDETERMINED and provide a note: `else: rule_status = "UNDETERMINED"; rule_note = "Uniform Energy Factor is required but not provided"`
+      - else, if the storage tank is greater than 55 and less than 100 gallons, Appendix F provides a heat pump efficiency, which may be an error.  return UNDETERMINED: `elif storage_volume <=100 and storage_volume > 55:`
+        - set rule_status to UNDETERMINED: `rule_status = "UNDETERMINED"`
+        - include a note: `rule_note = "The storage tank is between 55 and 100 gallons with a capacity <= 12kW (40945 btu/hr).  Appendix F provides a heat pump efficiency for water heaters in this capacity range, but the RCT does not have a way to verify this efficiency."
```

Comment:
I am not sure about this note. "The storage tank is between 55 and 100 gallons with a capacity <= 12kW (40945 btu/hr).  The minimum efficiency requirements in 90.1 Section 7.4.2 point to ASHRAE 90.1 Appendix F which provides a heat pump efficiency for water heaters in this capacity range which is not consistent with the efficiency of an electric resistance storage water heater which is the only electric SWH system type associated with the baseline for ASHRAE 90.1 Appendix G. Consequently, this rule was not able to be assessed for this service water heater. "